components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.scss

```diff
@@ -23,8 +36,13 @@
     top: 0;
   }
 
-  button:hover,
-  button:focus {
+  .failure-container button:hover,
+  .failure-container button:focus {
+    background-color: $chef-critical-hover;
+  }
+
+  .normal-container button:hover,
+  .normal-container button:focus {
     background-color: $chef-primary-hover;
```

Comment:
Besides the background color, we also want the hover and focus colors to adjust.